examples/bond/test/bond-token-flow.js

```diff
@@ -107,7 +107,7 @@ describe('Bond token Tests', function () {
       TMPL_APP_MANAGER: appManager.address
     };
     const issuerLsigProg = getProgram('issuer-lsig.py', scInitParam);
-    lsig = runtime.getLogicSig(issuerLsigProg, []);
+    lsig = runtime.createLSigAccount(issuerLsigProg, []);
```

Comment:
just to confirm, `LSig` <--> `Lsig`.

Reply:
Good catch, in other places we are using `Lsig`, so I will rename it here as well.

Reply:
renamed.